packages/components/src/custom-select-control-v2/test/index.tsx

```diff
@@ -416,4 +416,23 @@ describe.each( [
 			screen.getByRole( 'option', { name: 'july-9' } )
 		).toBeVisible();
 	} );
+
+	it( 'Should open the select popover when focussing the trigger button and pressing arrow down', async () => {
+		render( <Component { ...defaultProps } /> );
+
+		const currentSelectedItem = screen.getByRole( 'combobox', {
+			expanded: false,
+		} );
+
+		await press.Tab();
+		expect( currentSelectedItem ).toHaveFocus();
+		expect( currentSelectedItem ).toHaveTextContent( items[ 0 ].value );
+
+		await press.ArrowDown();
+		expect(
+			screen.getByRole( 'listbox', {
+				name: 'label!',
```

Comment:
Nit: should we use:

```suggestion
				name: defaultProps.label,
```

like we do for other text fixtures in tests?

Reply:
Done in this specific instance and throughout the three suites of tests https://github.com/WordPress/gutenberg/pull/62919/commits/c2f7c5917ee7e4b78f6a628ae9993cdce60295f6